axiom/tokens.go

```diff
@@ -0,0 +1,239 @@
+	// ID is the unique ID of the token.
+	ID string `json:"id"`
+	// Name is the name of the token.
+	Name string `json:"name"`
+	// Description is the description of the token.
+	Description string `json:"description"`
+	// ExpiresAt is the time when the token expires.
+	ExpiresAt time.Time `json:"expiresAt"`
+	// DatasetCapabilities is a map of dataset names to the capabilities available to that dataset for the token.
+	DatasetCapabilities map[string]DatasetCapabilities `json:"datasetCapabilities"`
+	// OrganisationCapabilities is the organisation capabilities available to the token.
+	OrganisationCapabilities OrganisationCapabilities `json:"orgCapabilities"`
+}
+
+// DatasetCapabilities represents the capabilities available to a token for a dataset.
+type DatasetCapabilities struct {
+	// Ingest is the ingest capability and the actions that can be performed on them.
+	Ingest []Action `json:"ingest"`
+	// Query is the query capability and the actions that can be performed on them.
+	Query []Action `json:"query"`
+	// StarredQueries is the starred queries capability and the actions that can be performed on them.
+	StarredQueries []Action `json:"starredQueries"`
+	// VirtualFields is the VirtualFields capability and the actions that can be performed on them.
+	VirtualFields []Action `json:"virtualFields"`
+}
+
+// OrganisationCapabilities represents the capabilities available to a token for an organisation.
+type OrganisationCapabilities struct {
+	// Annotations is the Annotations capability and the actions that can be performed on them.
+	Annotations []Action `json:"annotations,omitempty"`
+	// APITokens is the APITokens capability and the actions that can be performed on them.
+	APITokens []Action `json:"apiTokens,omitempty"`
+	// Billing is the Billing capability and the actions that can be performed on them.
+	Billing []Action `json:"billing,omitempty"`
+	// Dashboards is the Dashboards capability and the actions that can be performed on them.
+	Dashboards []Action `json:"dashboards,omitempty"`
+	// Datasets is the Datasets capability and the actions that can be performed on them.
+	Datasets []Action `json:"datasets,omitempty"`
+	// Endpoints is the Endpoints capability and the actions that can be performed on them.
+	Endpoints []Action `json:"endpoints,omitempty"`
+	// Flows is the Flows capability and the actions that can be performed on them.
+	Flows []Action `json:"flows,omitempty"`
+	// Integrations is the Integrations capability and the actions that can be performed on them.
+	Integrations []Action `json:"integrations,omitempty"`
+	// Monitors is the Monitors capability and the actions that can be performed on them.
+	Monitors []Action `json:"monitors,omitempty"`
+	// Notifiers is the Notifiers capability and the actions that can be performed on them.
+	Notifiers []Action `json:"notifiers,omitempty"`
+	// Rbac is the Rbac capability and the actions that can be performed on them.
+	Rbac []Action `json:"rbac,omitempty"`
```

Comment:
```suggestion
	RBAC []Action `json:"rbac,omitempty"`
```

Needs some changes, elsewhere, as well.